### ISSUE TITLE
Add proper header classes to h1/2/3/4/5 tags to help screen reading 

### ISSUE THREAD

**Issue by sandykadam** [visual-framework/vf-wp#1173]
Based on UX team accessibility review our websites which are using VF components are missing the proper heading classes which creates problem for screen readers.

e.g <h1 class=“vf-intro__heading “>Research groups</h1> this has H1 tag but we are missing its css class `vf-text-heading--1`
This applicable to all `<h>` tags across all our components.

It is there in our VF site documentation https://stable.visual-framework.dev/components/vf-heading/index.html?q=heading but it is missing in our respective components 

This ticket is to add all those missing classes for all h1/2/3/4/5 tags based on their css.

cc: @nikiforosk

**Comment by kasprzyk-sz** [comment]
When adding a component as a WP block I always follow the template from VF components documentation. There is no extra class for headings in `vf-intro`:

https://stable.visual-framework.dev/components/vf-intro/

In this case that should be updated there as well along with the YAML files which we use on static pages

**Comment by kasprzyk-sz** [comment]
Is there a way to solve it in the VF core instead of adding extra classes to every <h> element?

**Comment by sandykadam** [comment]
Yeh I don't see anything for intro but this is more for the headings used in other places, the one you have in this PR https://github.com/visual-framework/vf-wp/pull/1175 for now add on these.
